openssl/src/dh.rs

```diff
@@ -67,11 +111,13 @@ impl Dh {
     }
 }
 
+/// sets rust-openssl to call DH_set0_pqg as defined by fn from_params if using openSSL 1.1.x
```

Comment:
These docs won't be rendered anywhere since this is a private module, FYI.

Reply:
Okay, pulled.